src/Worker.php

```diff
@@ -95,11 +95,12 @@ protected function tick($queue)
         try {
             $this->event->acknowledge($command);
             $this->command_bus->handle($command);
-            $this->driver->processed($job);
             $this->event->finish($command);
```

Comment:
Do you think it would be valuable to move finished down to the `finally` too? Could maybe even send the handler response?